stable/neo4j/values.yaml

```diff
@@ -47,18 +47,34 @@ core:
     ##
     subPath: ""
 
-    ## Pass extra environment variables to the Neo4j container.
-    ##
-    # extraVars:
-    # - name: EXTRA_VAR_1
-    #   value: extra-var-value-1
-    # - name: EXTRA_VAR_2
-    #   value: extra-var-value-2
+  ## Pass extra environment variables to the Neo4j container.
+  ##
+  extraVars:
+  - name: EXTRA_VAR_1
+    value: extra-var-value-1
+  - name: EXTRA_VAR_2
+    value: extra-var-value-2
```

Comment:
This block should probably stay commented out.

Reply:
yeh good call. Uncommented by accident